test/functional/apps/dashboard/group1/url_field_formatter.ts

```diff
@@ -36,7 +37,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
   };
 
   // FLAKY: https://github.com/elastic/kibana/issues/133973
```

Comment:
Should we remove this Flaky comment if the test is no longer flaky?

Reply:
Yep. Will do.